test/fika/parser_test.exs

```diff
@@ -696,4 +696,51 @@ defmodule Fika.ParserTest do
       assert {:error, _, _, _, _, _} = Parser.expression(str)
     end
   end
+
+  describe "logic operators" do
+    test "supports consts" do
+      assert {:boolean, _line, true} = Parser.expression!("true")
```

Comment:
We already have this under boolean. Do we need this here?

Reply:
I was doing my tests and fogot, gotta remove